packages/server/src/async.ts

```diff
@@ -10,7 +10,9 @@ import { NextFunction, Request, RequestHandler, Response } from 'express';
  * @returns Async wrapped handler.
  */
 export function asyncWrap(callback: (req: Request, res: Response, next: NextFunction) => Promise<any>): RequestHandler {
-  return function (req: Request, res: Response, next: NextFunction): void {
+  const fn = function (req: Request, res: Response, next: NextFunction): void {
     callback(req, res, next).catch(next);
   };
+  Object.defineProperty(fn, 'name', { writable: false, value: callback.name });
```

Comment:
Could you add a comment explaining what's going on here? Namely that this is for the sake of traces so that the name of the `callback` parameter will be used?